ui/src/configureStore.js

```diff
@@ -0,0 +1,17 @@
+import { createStore, applyMiddleware } from 'redux'
+import thunkMiddleware from 'redux-thunk'
+import { createLogger } from 'redux-logger'
+import rootReducer from './reducers'
+
+let middleware = [ thunkMiddleware ]
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
Is `NODE_ENV` available in the frontend?

Reply:
Yes, it is added automatically by react.